### DISCUSSION TITLE
How to remove commas when rendering arrays in DataTable?

### DISCUSSION THREAD

**Discussion by avolutions** [mdbootstrap/TW-Elements#2044] [Support from community] [answered]
I'm using DataTable to render some values. One of the column is an array:
```
const data = {
  columns: ['Foo'],
  rows: [
    [ [ 1, 2, 3 ] ],
    [ [ 4, 5, 6 ] ],
  ],
};
```

It seems that arrays are automatically rendered as a comma separated string in DataTables:
![image](https://github.com/mdbootstrap/TW-Elements/assets/70471235/d2a0a5eb-fc82-44eb-8640-9c857fa0ac30)

Is there a way to provide another separator, e.g. if I want to use `|` or a space like this:
![image](https://github.com/mdbootstrap/TW-Elements/assets/70471235/449a9650-05ad-4ca5-900d-c8ea81d9dbbd)

**Comment by juujisai** [comment, marked as answer]
If you are trying to show the data in one column as shown on the screenshot, you can either provide the data as a string with separator of your choice or write a function that would change it for you

```
const formatData = (data) => data.toString().replaceAll(",", " | ");

const basicData = {
  columns: ["Foo"],
  rows: [[formatData([1, 2, 3])], [[4, 5, 6]]],
};
```
